pkg/flow/deduper.go

```diff
@@ -64,8 +58,8 @@ func Dedupe(expireTime time.Duration, justMark bool) func(in <-chan []*Record, o
 
 // isDupe returns whether the passed record has been already checked for duplicate for
 // another interface
-func (c *deduperCache) isDupe(key *ebpf.BpfFlowId) bool {
-	rk := *key
+func (c *deduperCache) isDupe(r *Record, justMark bool, fwd *[]*Record) {
```

Comment:
this function should be renamed as it doesn't return bool anymore, maybe something like `checkDupe` ?

Reply:
renamed